src/test/java/step03/domain/LottoTest.java

```diff
@@ -0,0 +1,133 @@
+package step03.domain;
+
+import exception.DuplicatedLottoNumberException;
+import exception.InValidSizeOfLottoException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import step01.Int;
+import step02.domain.WinningCounter;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class LottoTest {
+    List<LottoBall> lottoBalls;
+
+    @BeforeEach
+    void setup() {
+        lottoBalls = Arrays.asList(
+                LottoBall.valueOf(8),
+                LottoBall.valueOf(21),
+                LottoBall.valueOf(23),
+                LottoBall.valueOf(41),
+                LottoBall.valueOf(42),
+                LottoBall.valueOf(43)
+        );
+    }
+
+    @DisplayName("생성자")
+    @Test
+    void test_constructor() {
+        assertThat(Lotto.of(lottoBalls)).isEqualTo(Lotto.of(lottoBalls));
+    }
+
+    private static Stream<Arguments> provideTargetLottoResult() {
+        return Stream.of(
+                Arguments.of(Lotto.intOf(Arrays.asList(8, 1, 2, 3, 4, 5)), 1),
```

Comment:
```suggestion
                Arguments.of(Lotto.of(8, 1, 2, 3, 4, 5), 1),
```
이렇게 사용할 수 있도록 만들어보는건 어떨까요?